spec/convertion_by_format/check_open_docx_by_screen_spec.rb

```diff
@@ -17,6 +17,7 @@
       response = converter.perform_convert(url: uri.normalize.to_s, outputtype: 'png')
       expect(response[:url].nil?).to be_falsey
       expect(response[:url].empty?).to be_falsey
+      expect(ImageHelper.get_image_size(response[:url]) > StaticData::MIN_DOCX_IMAGE_SIZE).to be_truthy
```

Comment:
Better use rspec matchers like
`expect(ImageHelper.get_image_size(response[:url])).to be > (StaticData::MIN_DOCX_IMAGE_SIZE)`

If test fails - error will be more readable, compare output of 

`expect(1 > 2).to be_truthy`
```
expected: truthy value
            got: false
```
and 

`expect(1).to be > (2)`
```
Failure/Error: expect(1).to be > (2)

       expected: > 2
            got:   1
```